Filter/StringFilter.php

```diff
@@ -55,7 +55,7 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
      *
      * @return bool
      */
-    private function getOperator($type)
+    protected function getOperator($type)
```

Comment:
Also, one could imagine that there are several RDBMS in use in the project, so there should either be several instances of the String filter, or the string filter should be able to behave differently based on the filtered entity (the entity manager should be found, and the connection retrieved from it)